.github/workflows/publish-site.yml

```diff
@@ -0,0 +1,59 @@
+name: Publish Armeria site
+
+on:
+  push:
+    tags:
+      - armeria-*
+
+env:
+  LC_ALL: "en_US.UTF-8"
+
+jobs:
+  publish-site:
+    if: github.repository == 'line/armeria'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install svgbob_cli
+        run: |
+          sudo apt-get -y install cargo && cargo install svgbob_cli
+          echo "$HOME/.cargo/bin" >> $GITHUB_PATH
+
+      - id: setup-jdk-17
+        name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Restore the cache
```

Comment:
We tried to use `burrunan/gradle-cache-action` but reverted because we had encountered unknown build errors on Windows #3490
But the official Gradle action looks promising. Let me try to apply for this job.
If it works well, I'd also like to apply for our other jobs gradually.